app/Models/AssetMaintenance.php

```diff
@@ -150,6 +152,16 @@ public function admin()
         return $this->belongsTo(\App\Models\User::class, 'user_id')
             ->withTrashed();
     }
+    public function assignedTo()
```

Comment:
What is the difference between assignedTo and repairer here? Wouldn't the repairer be the person the maintenance is assigned to?

Reply:
The assignedTo targets who the asset belongs to. Not responsible for maintenance, but who it would be returned to when maintenance is done. I'm not opposed to just having the repairer be the assignedTo for this.

Reply:
might be less confusing.